vendor/modules.txt

```diff
@@ -1,4 +1,4 @@
-# github.com/dave/jennifer v1.4.1
+# github.com/dave/jennifer v1.4.2-0.20211112003305-45cc0b7eb71a
```

Comment:
Current commit on develop, which includes https://github.com/palantir/conjure-go/pull/241 (I don't think Dave is tagging releases anymore)